OpenRA.Mods.Common/Widgets/Logic/Lobby/SpawnSelectorTooltipLogic.cs

```diff
@@ -75,7 +87,8 @@ public SpawnSelectorTooltipLogic(Widget widget, TooltipContainerWidget tooltipCo
 			flag.IsVisible = () => playerFaction != null;
 			flag.GetImageCollection = () => "flags";
 			flag.GetImageName = () => playerFaction;
-			team.GetText = () => $"Team {playerTeam}";
+			var teamMessage = modData.Translation.GetString(Team, Translation.Arguments("team", playerTeam));
+			team.GetText = () => teamMessage;
```

Comment:
The value of `playerTeam` is always `-1` at this point so this message will be stuck with that. It needs to be updated inside the `beforeRender` callback.
![image](https://user-images.githubusercontent.com/1355810/185666526-17a3a94a-57c3-4d46-8ae9-fc53839cee46.png)

Reply:
Yes, this needs another `CachedTransform`.